service/grpc/codelingo/codelingo.pb.go

```diff
@@ -108,20 +80,6 @@ func (m *DescribeFactReply) String() string            { return proto.CompactTex
 func (*DescribeFactReply) ProtoMessage()               {}
```

Comment:
exported method DescribeFactReply.Descriptor should have comment or be unexported